app/models/conference.rb

```diff
@@ -113,6 +113,24 @@ def confirmed
   before_create :create_email_settings
   before_create :add_color
 
+  def date_range_string
+    startstr = 'Unknown - '
+    endstr = 'Unknown'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.